example/service/dynamodb/scanItems/README.md

```diff
@@ -10,9 +10,10 @@ The `Item` time will be used by the example to unmarshal the DynamoDB table's it
 type Item struct {
 	Key  int
 	Desc string
-	Data map[string]interface{}
+	Data map[string]interface{} `dynamodbav:"Data,omitempty"`
 }
 ```
+Use Go tags to define what the name is of the attribute in your DynamoDB table. See https://godoc.org/github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute#Marshal for more information.
```

Comment:
Should point to our docs - http://docs.aws.amazon.com/sdk-for-go/api/service/dynamodb/dynamodbattribute/#Marshal